modin/experimental/sql/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import warnings
+
+try:
+    import dfsql
+    from dfsql import sql_query
+    import dfsql.extensions
+except ImportError:
+    warnings.warn(
```

Comment:
I prefer to throw our own error message and not rethrow the `ImportError`. I think it might give too many error messages if we add an extra warning.

Reply:
I haven't found a fitting custom exception class in Modin. What should I use to throw the error?

Reply:
You can just raise an `ImportError`.

Reply:
Maybe remove the `try except` altogether then? Not very Pythonic to override an `ImportError` with custom text, if I understood your meaning correctly

Reply:
Let's just leave it as you have it. I think it will be ok.